### PR TITLE
Update ODBC user manual for signed ODBC driver

### PR DESCRIPTION
**Pull request by chloe-zh**
*Issue #, if available:*

*Description of changes:*
- Original instructions require users to set **Options and settings - Options - Security - Data Extensions** to **(Not Recommended) Allow any extension to load without validation or warning**. Removed this instruction in this PR.
- After install the ODBC driver in Windows host, the ODBC Data Source Administrator also sets a DSN in the system DSN, localhost:9200 as the host and no authentication by default. Added this setting option in the user manual in this PR.
- Both import and query methods are supported to load data in the latest PowerBI now, added this option in the user manual.
- Updated the images.

Preview: https://github.com/chloe-zh/sql/blob/odbc-doc/sql-odbc/docs/user/power_bi_support.md 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: sql-odbc/docs/user/power_bi_support.md [hunk 1]
```diff
@@ -8,39 +8,33 @@
...
 
 ## Setup
-* Copy `OdfeSqlOdbcPBIConnector.mez` file in the `<User>\Documents\Power BI Desktop\Custom Connectors\` folder. This will let Power BI access custom connector.
-* Open Power BI Desktop.
-* Change the security settings. Click on **Files** > **Options and settings** > **Options** > **Security** > Select **Allow any extension to load without validation or warning** for Data Extensions. This will allow the custom connector to load data into Power BI.
-
-<img src="img/pbi_settings.png" width="500">
-
-* Restart Power BI Desktop.
+* Copy `OdfeSqlOdbcPBIConnector.mez` file in the `<User>\Documents\Power BI Desktop\Custom Connectors\` folder. This will let Power BI access custom connector. If you have installed the On-premises data gateway, you should be able to see the ODBC is detected as a custom connector in the **Connectors** tab.
```
Comment (penghuo) [inline sql-odbc/docs/user/power_bi_support.md]: 1. Copy `OdfeSqlOdbcPBIConnector.mez` file to
2. Could we use the msi to install the mez?
3. Does the mez need the release procedure, how do we maintain it in furutre?

**Comment by penghuo** [comment]
The download link doesn't contain any packages. Please link the correct pages.
```
If you get an following error, please install Open Distro For Elasticsearch SQL ODBC Driver.
```

**Review by anirudha — APPROVED** [review]
Thanks

**Review by dai-chen — APPROVED** [review]

File: sql-odbc/docs/user/power_bi_support.md (hunk 1 above)
Comment (chloe-zh) [reply sql-odbc/docs/user/power_bi_support.md]: Corrected the typo. For the mez file, we need more discussion over it offline, let's keep it out of scope for this pr.

**Comment by chloe-zh** [comment]
> The download link doesn't contain any packages. Please link the correct pages.
> 
> ```
> If you get an following error, please install Open Distro For Elasticsearch SQL ODBC Driver.
> ```

Thanks, replaced the link with the connector download page https://opendistro.github.io/for-elasticsearch/downloads.html#connect

**Review by penghuo — APPROVED** [review]

**Pull request merged**